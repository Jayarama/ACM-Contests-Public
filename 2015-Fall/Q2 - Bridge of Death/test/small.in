Arthur, Bedemir
What is your name?
What is your quest?
What is your favorite colour?
What is your name?
What is your quest?
What is your favorite colour?
Robin
What is your name?
What is your quest?
What is your favorite color?
Galahad, Lancelot
What is your name?
What is your quest?
What is your favorite color?
What is your name?
What is your quest?
What is your favorite color?
