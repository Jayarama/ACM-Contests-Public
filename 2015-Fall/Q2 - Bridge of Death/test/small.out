Arthur, King of the Britons.
I seek the grail.
Gold.
Sir Bedevere of Camelot.
I seek the grail.
Blue.
Sir Robin of Camelot.
I seek the grail.
Yellow.
Sir Galahad of Camelot.
I seek the grail.
Yellow.
Sir Lancelot of Camelot.
I seek the grail.
Blue.
